src/main/java/com/kodilla/ecommercee/service/CartService.java

```diff
@@ -35,28 +34,28 @@
 
 
     public Cart createCart(Cart cart) {
-        return cartRepository.save(cart);
+        return cartRepository.save(new Cart());
     }
 
     public Item getItem(Long itemId) {
         return itemRepository.getOne(itemId);
     }
 
-    public boolean addItem(Item item, Long cartId) {
-       return cartRepository.getOne(cartId).getItems().add(itemRepository.save(item));
+    public boolean addItem(Item item) {
+       return cartRepository.getOne(item.getCart().getId()).getItems().add(itemRepository.save(item));
```

Comment:
To niepotrzebne. Przed zapisem item trzeba sprawdzić, czy istnieje id Cart i Product -> jak tak, to można zapisać